scripts/test_schemas.sh

```diff
@@ -17,9 +17,6 @@ if [ "$result" != "{}" ] && [ "$result" != "" ]; then
 echo "---Schema Failed Validation---"
 echo "Error: [$result]"
 exit=1
-else
-echo "---Schema Passed Validation---"
```

Comment:
I like this change, its just the exit code that was causing it to fail

Reply:
Ya, so, for now, I removed it to see if that was the issue and turns out it was.